stable/kommander/values.yaml

```diff
@@ -1,8 +1,10 @@
 image:
   repository: mesosphere/kommander
-  tag: 1.40.0
+  tag: 1.45.0
   pullPolicy: IfNotPresent
 replicas: 1
+isKonvoyUI: false
+logoutRedirectPath: "/"
```

Comment:
also, is this a default for kommander or it is only used for konvoy-ui too ?

Reply:
@DanielMSchmidt ☝️

Reply:
I am unsure, it's the default we have in the backend code as well, so I think it's a save bet. @natmegs or @brandonc might know more about ops portal related logout routing

Reply:
It's used in both